packages/labs/observers/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Change Log
+
+Changes below were based on the [Keep a Changelog](http://keepachangelog.com/) format. All changes above are generated automatically by [Changesets](https://github.com/atlassian/changesets).
```

Comment:
don't include this for a new changelog, since changesets doesn't follow this format

Reply:
Do we need this file at all?

Reply:
Maybe not? I think `changeset version` will create it.

Reply:
Removed.